packages/headless/src/controllers/core/triggers/headless-core-notify-trigger.ts

```diff
@@ -0,0 +1,21 @@
+import {Controller} from '../../controller/headless-controller';
+
+/**
+ * The `NotifyTrigger` controller handles notify triggers. A [Notify trigger] (https://docs.coveo.com/en/3413#notify) query pipeline rule lets you define a message to be displayed to the end user when a certain condition is met.
```

Comment:
I think these need to be next to eachother for the link to work
```suggestion
 * The `NotifyTrigger` controller handles notify triggers. A [Notify trigger](https://docs.coveo.com/en/3413#notify) query pipeline rule lets you define a message to be displayed to the end user when a certain condition is met.
```